insecure/attacker.proto

```diff
@@ -11,14 +11,10 @@ option go_package = "insecure";
 
 // CorruptibleConduitFactory allows an attacker to register itself to the factory, and send messages on behalf of its conduits.
 service CorruptibleConduitFactory {
-  rpc RegisterAttacker(AttackerRegisterMessage) returns (google.protobuf.Empty) {}
+  rpc RegisterAttacker(google.protobuf.Empty) returns (stream Message) {}
   rpc ProcessAttackerMessage(stream Message) returns (google.protobuf.Empty) {}
 }
 
-service Attacker {
-  rpc Observe(stream Message) returns (google.protobuf.Empty) {}
-}
-
```

Comment:
This is the core of changes to this PR. The rest are necessary refactoring to adapt this change. Basically, instead of having two separate gRPC interfaces for attacker and CCFs, we consolidated them into one interface, only for the CCF.